src/wmnds/components/content-tiles/page-contents/_page-contents.scss

```
@@ -0,0 +1,10 @@
+.wmnds-content-tile {
+  &__title--content {
+    color: get-color(text);
+  }
+
+  .overview {
```

Again make sure the class is more like...
```suggestion
  &__overview {
```
You can read more about why and how we use something called BEM CSS styling here: [https://github.com/wmcadigital/wmn-design-system/blob/master/doc/contributing/css.md](https://github.com/wmcadigital/wmn-design-system/blob/master/doc/contributing/css.md)